src/theme/index.hbs

```diff
@@ -85,9 +88,7 @@
             var html = document.querySelector('html');
             html.classList.remove('{{ default_theme }}')
             html.classList.add(theme);
-            var body = document.querySelector('body');
-            body.classList.remove('no-js')
-            body.classList.add('js');
+            document.body.classList.add("js");
```

Comment:
If you're using a `noscript.css` file, do you need this at all?

Reply:
For theme handling in case JS is actually enabled: you don't want the system theme to be applied in this case by default.

Reply:
Ah, I understand now. This is specifically used by `variables.css`.